### ISSUE TITLE
Election date and election type should flow to end-of-round page.

### ISSUE THREAD

**Issue by sfsinger19103** [FreeAndFair/ColoradoRLA#632]
After completing the first round of the audit the election date and election type are not entered correctly.

![image](https://user-images.githubusercontent.com/14353822/29988364-bc2f7636-8f21-11e7-9673-9b5b2037db15.png)

**Comment by dmzimmerman** [comment]
This is still the case, but now it shows up as:

```
Congratulations! You have completed reporting the votes on all ballots randomly selected for this round of the risk-limiting audit of the Adams County Invalid date Election .
```

**Comment by sfsinger19103** [comment]
Simply changing the text to omit mention of the election and the date would be fine.

**Comment by ranweiler** [comment]
This was fixed in #702, specifically, ae5fb59f3c8bfd244f921b80670b96d17dd8badf.
